### ISSUE TITLE
Another KaTeX Rendering Bug

### ISSUE THREAD

**Issue by rdipietro** [notable/notable#448]
- **OS Version**: MacOS High Sierra
- **Notable Version**: 1.3.0

### Current behavior

```
\mathbb{V} = \mathbb{V}_1 + \mathbb{V}_2 + \cdots + \mathbb{V}_{1,2} + \cdots
```

renders as

![image](https://user-images.githubusercontent.com/5150559/52650468-4bad0080-2eb8-11e9-8b77-3d6293817e12.png)

### Expected behavior

Should render as 
![image](https://user-images.githubusercontent.com/5150559/52650394-2ddf9b80-2eb8-11e9-830d-ca19dd617a86.png)

(This can be verified by pasting into the example box at https://katex.org.)

**Comment by fabiospampinato** [comment]
Apparently this is only rendered properly inside a codeblock.

<img width="685" alt="screen shot 2019-02-12 at 17 26 30" src="https://user-images.githubusercontent.com/1812093/52650881-60ef5280-2eeb-11e9-8a4a-b66897583e8b.png">

**Comment by fabiospampinato** [comment]
Fixed in v1.6.
